diff --git forkSrcPrefix/generated/BondPurchase_types.ts forkDstPrefix/generated/BondPurchase_types.ts
index 7084aec01f2535ca934ccaf70e88edd19c49de43..320da1c2c10abc0cb88aca535fa642583ccb9cfb 100644
--- forkSrcPrefix/generated/BondPurchase_types.ts
+++ forkDstPrefix/generated/BondPurchase_types.ts
@@ -232,7 +232,6 @@ export type Market = {
 export type MarketClosedEvent = {
   __typename?: 'MarketClosedEvent';
   id: Scalars['ID'];
-  marketId: Scalars['BigInt'];
   bondContract: Scalars['Bytes'];
   bondType: BondType;
   date: Scalars['String'];
@@ -250,14 +249,6 @@ export type MarketClosedEvent_Filter = {
   id_lte?: InputMaybe<Scalars['ID']>;
   id_in?: InputMaybe<Array<Scalars['ID']>>;
   id_not_in?: InputMaybe<Array<Scalars['ID']>>;
-  marketId?: InputMaybe<Scalars['BigInt']>;
-  marketId_not?: InputMaybe<Scalars['BigInt']>;
-  marketId_gt?: InputMaybe<Scalars['BigInt']>;
-  marketId_lt?: InputMaybe<Scalars['BigInt']>;
-  marketId_gte?: InputMaybe<Scalars['BigInt']>;
-  marketId_lte?: InputMaybe<Scalars['BigInt']>;
-  marketId_in?: InputMaybe<Array<Scalars['BigInt']>>;
-  marketId_not_in?: InputMaybe<Array<Scalars['BigInt']>>;
   bondContract?: InputMaybe<Scalars['Bytes']>;
   bondContract_not?: InputMaybe<Scalars['Bytes']>;
   bondContract_in?: InputMaybe<Array<Scalars['Bytes']>>;
@@ -331,7 +322,6 @@ export type MarketClosedEvent_Filter = {
 
 export enum MarketClosedEvent_OrderBy {
   Id = 'id',
-  MarketId = 'marketId',
   BondContract = 'bondContract',
   BondType = 'bondType',
   Date = 'date',
@@ -343,7 +333,6 @@ export enum MarketClosedEvent_OrderBy {
 export type MarketCreatedEvent = {
   __typename?: 'MarketCreatedEvent';
   id: Scalars['ID'];
-  marketId: Scalars['BigInt'];
   bondContract: Scalars['Bytes'];
   bondType: BondType;
   date: Scalars['String'];
@@ -361,14 +350,6 @@ export type MarketCreatedEvent_Filter = {
   id_lte?: InputMaybe<Scalars['ID']>;
   id_in?: InputMaybe<Array<Scalars['ID']>>;
   id_not_in?: InputMaybe<Array<Scalars['ID']>>;
-  marketId?: InputMaybe<Scalars['BigInt']>;
-  marketId_not?: InputMaybe<Scalars['BigInt']>;
-  marketId_gt?: InputMaybe<Scalars['BigInt']>;
-  marketId_lt?: InputMaybe<Scalars['BigInt']>;
-  marketId_gte?: InputMaybe<Scalars['BigInt']>;
-  marketId_lte?: InputMaybe<Scalars['BigInt']>;
-  marketId_in?: InputMaybe<Array<Scalars['BigInt']>>;
-  marketId_not_in?: InputMaybe<Array<Scalars['BigInt']>>;
   bondContract?: InputMaybe<Scalars['Bytes']>;
   bondContract_not?: InputMaybe<Scalars['Bytes']>;
   bondContract_in?: InputMaybe<Array<Scalars['Bytes']>>;
@@ -442,7 +423,6 @@ export type MarketCreatedEvent_Filter = {
 
 export enum MarketCreatedEvent_OrderBy {
   Id = 'id',
-  MarketId = 'marketId',
   BondContract = 'bondContract',
   BondType = 'bondType',
   Date = 'date',
@@ -460,14 +440,6 @@ export type Market_Filter = {
   id_lte?: InputMaybe<Scalars['ID']>;
   id_in?: InputMaybe<Array<Scalars['ID']>>;
   id_not_in?: InputMaybe<Array<Scalars['ID']>>;
-  marketId?: InputMaybe<Scalars['BigInt']>;
-  marketId_not?: InputMaybe<Scalars['BigInt']>;
-  marketId_gt?: InputMaybe<Scalars['BigInt']>;
-  marketId_lt?: InputMaybe<Scalars['BigInt']>;
-  marketId_gte?: InputMaybe<Scalars['BigInt']>;
-  marketId_lte?: InputMaybe<Scalars['BigInt']>;
-  marketId_in?: InputMaybe<Array<Scalars['BigInt']>>;
-  marketId_not_in?: InputMaybe<Array<Scalars['BigInt']>>;
   bondContract?: InputMaybe<Scalars['Bytes']>;
   bondContract_not?: InputMaybe<Scalars['Bytes']>;
   bondContract_in?: InputMaybe<Array<Scalars['Bytes']>>;
